CHANGELOG.md

```diff
@@ -66,13 +66,12 @@
 
 ### Fixes
 
-#### PDK
-
-- `pdk.response.set_header()`, `pdk.response.set_headers()`, `pdk.response.exit()` now ignore and emit warnings for manually set `Transfer-Encoding` headers.
-  [#8698](https://github.com/Kong/kong/pull/8698)
-
 ### Breaking Changes
 
+- Deprecate/stop producing Amazon Linux (1) containers and packages (EOLed December 31, 2020)
+  [Kong/kong-ee #3247](https://github.com/Kong/kong-ee/pull/3247)
```

Comment:
We should not link to kong-ee and kong-distributions here, since those are private. Typically, only a link to the kong/kong PR is added.

Reply:
✅